types/nouislider/index.d.ts

```diff
@@ -4,195 +4,196 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 /// <reference types="wnumb"/>
 
-export = noUiSlider;
-export as namespace noUiSlider;
```

Comment:
You're right. I've added back the UMD declaration.

I also took the chance to add a missing option and use literal types where necessary ;)